javascript/eslint.md

```diff
@@ -1,37 +1,37 @@
 
-# Code liniting + ESLint Instructions
+# Linting with ESLint Manual
 
 ### Projected Time
 60 minutes
 
 - Lesson: 15-20 min
-- Guided Practice: 5 -10 min (Mock Project)
-- Independed Practice: 30min
+- Guided Practice: 5-10 min (Mock Project)
+- Independent Practice: 30min
 - Check for Understanding: 1 min
 
 ### Prerquisites
 
 Here are topics you must be comfortable with to get the maximum benefit out of this topic:
 
--  [Javascript 6 - Object Literals](https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-6-object-literals.md)
+- [Javascript 6-Object Literals](https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-6-object-literals.md)
```

Comment:
I'm not sure about this change.  Check out the md preview 
![image](https://user-images.githubusercontent.com/29131689/77855961-e34adc80-71a8-11ea-9b2f-60a46078a4bd.png)

Reply:
Oh also, the word "Prerquisites" needs help